.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,5 +5,35 @@ Thank you for your Pull Request!
 
 If you are new to contributing to Mono, please try to do your best at conforming to our coding guidelines http://www.mono-project.com/community/contributing/coding-guidelines/ but don't worry if you get something wrong. One of the project members will help you to get things landed.
```

Comment:
Should we also remove this top section? I'm not sure that it applies to changes in our fork.